### PR TITLE
Bug hunting 1

### LINKED ISSUE #409: Adding link to website to a menu item
In the "Static Content" page, add a new button (`Add link to website`) next to the `Add new static content` button. The user should be able to enter a URL (with or without http:// prefix)

This button must create a menu item with a `ServiceMenuItemLinkTO(external=False)`
See `link` in `system.put_menu_item`
Eg. "mijn bibliotheek" in sint-truiden app

Requested by @stevendepaepe in #402

### PR DESCRIPTION
**Pull request by lucasvanhalst**
Allows opening a website when clicking on a menu item.

Close #409

### CODE REVIEW

File: src-test/test/solutions/test_customer_store.py [hunk 1]
```diff
@@ -16,24 +16,24 @@
 # @@license_version:1.2@@
 
 import base64
+from test import set_current_user
```
Comment (FastGeert) [inline src-test/test/solutions/test_customer_store.py]: from test ?

Comment (FastGeert) [inline src-test/test/solutions/test_customer_store.py]: never mind, this is unit test code

File: src-test/test/solutions/test_customer_store.py [hunk 2]
```diff
@@ -102,6 +102,34 @@ def _create_service(self, customer):
...
+        service.current_user_app_infos = []
+        service.managed_organization_types = []
+
+        put_customer_service(customer, service, skip_module_check=True, search_enabled=False,
+                             skip_email_check=True, rollback=True)
+
+        put_customer_service(customer, service, skip_module_check=True, search_enabled=False,
+                             skip_email_check=True, rollback=True)
+
+
```
Comment (FastGeert) [inline src-test/test/solutions/test_customer_store.py]: Where is the test that asserts if things have been put successfully?

File: src/shop/bizz.py [hunk 3]
```diff
@@ -726,8 +726,10 @@ def put_service(customer_or_id, service, skip_module_check=False, search_enabled
                             search_enabled, qualified_identifier=service.email, broadcast_to_users=broadcast_to_users)
 
     # Setting service_email to make sure the auto_login_url can be constructed
-    customer.service_email = customer.service_email or service.email
+    old_service_email = customer.service_email
+    customer.service_email = u"tmp email"
     r.auto_login_url = customer.auto_login_url
+    customer.service_email = old_service_email
```
Comment (FastGeert) [inline src/shop/bizz.py]: always use a try finally in these kinds of code blocks

File: src-test/test/solutions/test_customer_store.py (hunk 2 above)
Comment (RubenMattan) [inline src-test/test/solutions/test_customer_store.py]: Not needed it will crash

File: src/shop/bizz.py (hunk 3 above)
Comment (RubenMattan) [inline src/shop/bizz.py]: The code can't fail

File: src/shop/bizz.py [hunk 4]
```diff
@@ -727,9 +727,11 @@ def put_service(customer_or_id, service, skip_module_check=False, search_enabled
...
     # Setting service_email to make sure the auto_login_url can be constructed
     old_service_email = customer.service_email
-    customer.service_email = u"tmp email"
-    r.auto_login_url = customer.auto_login_url
-    customer.service_email = old_service_email
+    try:
+        customer.service_email = u"tmp email"
+        r.auto_login_url = customer.auto_login_url
+    finally:
+        customer.service_email = old_service_email
```
Comment (lucasvanhalst) [inline src/shop/bizz.py]: What's the point of this

Comment (RubenMattan) [inline src/shop/bizz.py]: This fixes https://github.com/our-city-app/oca-backend/issues/422

**Pull request merged**